src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1726,6 +1726,9 @@ public IReadOnlyList<SyntaxNode> GetStatementContainerStatements(SyntaxNode? nod
                 return ImmutableArray<SyntaxNode>.Empty;
         }
 
+        public bool IsConversionExpression([NotNullWhen(true)] SyntaxNode? node)
+            => node is CastExpressionSyntax;
+
         public bool IsCastExpression([NotNullWhen(true)] SyntaxNode? node)
             => node is CastExpressionSyntax;
```

Comment:
Not sure why both are needed if they're doing the same thing?

Reply:
they're not hte same in VB.  C# merges tehse concepts, vb separates.